.vscode/launch.json

```diff
@@ -356,7 +356,7 @@
       "request": "launch",
       "cwd": "${workspaceRoot}",
       "program":
-        "${workspaceRoot}/packages/salesforcedx-vscode-replay-debugger/out/src/adapter/apexReplayDebug.js",
+        "${workspaceRoot}/packages/salesforcedx-vscode-apex-replay-debugger/out/src/adapter/apexReplayDebug.js",
```

Comment:
@DatGuyJonathan - I'm confused. You said that you're not changing the name of the folder yet but I see that you change the name here in launch.json so the folder name needs to change as well or else we won't be able to launch anymore?

Reply:
Oops, reverting.